### ISSUE TITLE
redis.exceptions.ResponseError: syntax error (set)

### ISSUE THREAD

**Issue by taochen-ct** [nalgeon/redka#29]
python 3.9

redis == 5.0.3
redka == 0.5.1

```
import redis

redis = redis.Redis(host='localhost', port=6379, db=0)

redis.set('my_key', 'my_value', nx=True, px=10)

value = redis.get('my_key')
print(value.decode())  # 输出：my_value
```

raise Exception redis.exceptions.ResponseError: syntax error (set)
because of not support  **nx** and **px**

**Comment by nalgeon** [comment]
Does not reproduce. Are you sure you are running Redka 0.5.1? How exactly did you install and run it?

**Comment by taochen-ct** [comment]
> Does not reproduce. Are you sure you are running Redka 0.5.1? How exactly did you install and run it?

![image](https://github.com/nalgeon/redka/assets/93983997/e1ecaad6-b72c-4656-9584-af06b74bc3bd)
this method on macos
run the redka in the console， ./redka app.db
the app.db also uesed by flask

**Comment by nalgeon** [comment]
Oh, I see. It was fixed in ce62018609915de3af7b06338566ac5c532296e1, which came after v0.5.1. Released now in [v0.5.2](https://github.com/nalgeon/redka/releases/v0.5.2).
